packages/elements/src/utils/guards.ts

```diff
@@ -1,8 +1,8 @@
 import { IHttpOperation, IHttpRequest, IHttpService } from '@stoplight/types';
-import { JSONSchema4 } from 'json-schema';
+import { JSONSchema7 } from 'json-schema';
 import { isObject } from 'lodash';
 
-export function isJSONSchema(maybeSchema: unknown): maybeSchema is JSONSchema4 {
+export function isJSONSchema(maybeSchema: unknown): maybeSchema is JSONSchema7 {
```

Comment:
Type safety FTW 🔒 😂

Reply:
:rofl: I have to admit I love these guards.